test/Riok.Mapperly.IntegrationTests/Riok.Mapperly.IntegrationTests.csproj

```diff
@@ -44,4 +44,10 @@
       <IncludeAssets>runtime; build; native; contentfiles; analyzers</IncludeAssets>
     </PackageReference>
   </ItemGroup>
+  <ItemGroup>

```

Comment:
Is this needed? iirc I'd accidentally add a similar block when deleting unneeded tests.